libs/model/src/services/commonProtocol/contestHelper.ts

```diff
@@ -0,0 +1,135 @@
+import { AppError } from '@hicommonwealth/core';
+import Web3 from 'web3';
+import { AbiItem } from 'web3-utils';
+import { contestABI } from './abi/contestAbi';
+
+export type AddContentResponse = {
+  txReceipt: any;
+  contentId: string;
+};
+
+export type ContestStatus = {
+  startTime: number;
+  endTime: number;
+  contestInterval: number;
+  currWinners: string[];
+  lastContentId: string;
+};
+
+/**
+ * Adds content to an active contest. Includes validation of contest state
+ * @param web3 an instance of web3.js with correct RPC and Private Key
+ * @param contest an instance of web3.js with correct RPC and Private Key
```

Comment:
Please fix the description of the `contest` param– judging by usage, I'm guessing it's the address of the contest contract.